src/Middleware/HttpOverrides/src/ForwardedHeadersOptions.cs

```diff
@@ -83,7 +84,10 @@ public class ForwardedHeadersOptions
     /// <summary>
     /// Address ranges of known proxies to accept forwarded headers from.
     /// </summary>
-    public IList<IPNetwork> KnownNetworks { get; } = new List<IPNetwork>() { new IPNetwork(IPAddress.Loopback, 8) };
+    public IList<IPNetwork> KnownNetworks { get; } = new List<IPNetwork>()
```

Comment:
This change won't be necessary once the runtime is updated, right?

Reply:
Right, I think so, will revert in later update